grpc-all/1.68.0.wso2v1/META-INF/services/io.grpc.LoadBalancerProvider

```diff
@@ -0,0 +1,2 @@
+io.grpc.internal.PickFirstLoadBalancerProvider
```

Comment:
missing license header

Reply:
These are some class files. We cannot add license for them